demucs/spec.py

```diff
@@ -17,17 +17,7 @@ def spectro(x, n_fft=512, hop_length=None, pad=0, onnx_exportable=False):
         x = x.cpu()
     
     if onnx_exportable:
-        z = th.view_as_real(
-            th.stft(x,
-                    n_fft * (1 + pad),
-                    hop_length or n_fft // 4,
-                    window=th.hann_window(n_fft).to(x),
-                    win_length=n_fft,
-                    normalized=True,
-                    center=True,
-                    return_complex=True,
-                    pad_mode='reflect')
-        )   # Now z will return 1 more dimension - last dimension will be 2 now
+        z = demucs_stft(x.view(-1, 1, length))    # z will return 1 more dimension - z.size(-1) will be 2
```

Comment:
In case you follow the bellow suggestion, you should be able to keep pretty much the same API
```suggestion
        z = stft(x.view(-1, 1, length)),
                    n_fft * (1 + pad),
                    hop_length or n_fft // 4,
                    win_type=WindowsType.HANN,
                    win_length=n_fft,
                    normalized=True,
                    center=True,
                    pad_mode='reflect')
        )   # z will return 1 more dimension - z.size(-1) will be 2
```

Note: 
- `WindowsType` needs to be imported from `.stft`
- if `pad` > 0, this will fail your window check, so not sure if it is possible updating to just `n_fft` (instead of `n_fft * (1 + pad)`)

Reply:
same as above, this will not be modified